vitest.config.mts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'vitest/config';
+import { name } from './package.json';
+
+export default defineConfig({
+  test: {
+    name,
+    coverage: {
+      provider: 'istanbul',
+      include: ['src/**/*'],
+    },
+  },
+});
```

Comment:
Could we move this to the benchmark folder to make sure it's scope to it?

Reply:
I put it as the default setting for the bitest, but if you don't want to see it, you don't need it